helm-frontend/project_metadata.json

```diff
@@ -29,6 +29,12 @@
 		"id": "mmlu",
 		"releases": ["v1.1.0", "v1.0.0"]
 	},
+	{
+	  "title": "VHELM",
+	  "description": "Holistic evaluation of Vision-Language Models",
```

Comment:
Uppercase 'Evaluation' (or just be consistent with everything else)

Reply:
fixed